docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx
+sphinx-click
+fromager
```

Comment:
Missing newline at EOF
You also don't need to include the package as dependency. It's implied.